src/domain/usecase/CreateSpot.test.ts

```diff
@@ -0,0 +1,103 @@
+import { CreateSpot } from './CreateSpot'
+import { SpotRepository } from '../repository/SpotRepository'
+import { Spot, CreateSpotCommand } from '../model/Spot'
+import { Location } from '../model/Location'
+
+const createSpot = (createSpotCommand: CreateSpotCommand): Spot => ({
+  id: 'spot-id',
```

Comment:
Why pass an argument `createSpotCommand` when it is not used ?

Reply:
Indeed – removed in 03fc648967d6b15fe9919927df9df51315aa4c90.